src/hardware/input/mouse_config.cpp

```diff
@@ -638,13 +638,11 @@ static void init_mouse_config_settings(SectionProp& secprop)
 	prop_bool = secprop.AddBool("vmware_mouse", OnlyAtStart, true);
 	prop_bool->SetHelp(
 	        "VMware mouse interface ('on' by default).\n"
-	        "with experimental 3rd party Windows 3.1x driver.\n"
 	        "Note: Requires PS/2 mouse to be enabled.");
 
 	prop_bool = secprop.AddBool("virtualbox_mouse", OnlyAtStart, true);
 	prop_bool->SetHelp(
 	        "VirtualBox mouse interface ('on' by default).\n"
-	        "Fully compatible only with 3rd party Windows 3.1x driver.\n"
```

Comment:
You might want to restore these until we make Win9x support _official_. That involves not just the mouse, but all sorts of other stuff like CPU emulation.